tbconnect/models.py

```diff
@@ -0,0 +1,83 @@
+import uuid
+
+import pycountry
+from django.db import models
+
+from userprofile.validators import geographic_coordinate, za_phone_number
+
+
+class TBCheck(models.Model):
+    AGE_U18 = "<18"
+    AGE_18T40 = "18-40"
+    AGE_40T65 = "40-65"
+    AGE_O65 = ">65"
+    AGE_CHOICES = (
+        (AGE_U18, AGE_U18),
+        (AGE_18T40, AGE_18T40),
+        (AGE_40T65, AGE_40T65),
+        (AGE_O65, AGE_O65),
+    )
+
+    PROVINCE_CHOICES = sorted(
+        (s.code, s.name) for s in pycountry.subdivisions.get(country_code="ZA")
+    )
+
+    EXPOSURE_YES = "yes"
+    EXPOSURE_NO = "no"
+    EXPOSURE_NOT_SURE = "not_sure"
+    EXPOSURE_CHOICES = (
+        (EXPOSURE_YES, "Yes"),
+        (EXPOSURE_NO, "No"),
+        (EXPOSURE_NOT_SURE, "Not sure"),
+    )
+
+    COUGH_NO = "no"
+    COUCH_YES_LT_2WEEKS = "yes_lt_2weeks"
+    COUCH_YES_GT_2WEEKS = "yes_gt_2weeks"
+    COUGH_CHOICES = (
+        (COUGH_NO, "No"),
+        (COUCH_YES_LT_2WEEKS, "Yes, less than 2 weeks"),
+        (COUCH_YES_GT_2WEEKS, "Yes, more than 2 weeks"),
+    )
+
+    RISK_LOW = "low"
+    RISK_MODERATE_WITHOUT_COUGH = "moderate_without_cough"
+    RISK_MODERATE_WITH_COUGH = "moderate_with_cough"
+    RISK_HIGH = "high"
+    RISK_CHOICES = (
+        (RISK_LOW, "Low"),
+        (RISK_MODERATE_WITHOUT_COUGH, "Moderate without cough"),
+        (RISK_MODERATE_WITH_COUGH, "Moderate with cough"),
+        (RISK_HIGH, "High"),
+    )
+
+    GENDER_MALE = "male"
+    GENDER_FEMALE = "female"
+    GENDER_OTHER = "other"
+    GENDER_NOT_SAY = "not_say"
+    GENDER_CHOICES = (
+        (GENDER_MALE, "Male"),
+        (GENDER_FEMALE, "Female"),
+        (GENDER_OTHER, "Other"),
+        (GENDER_NOT_SAY, "Rather not say"),
+    )
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
I actually consider this a mistake on the original model. Pagination is much easier on normal auto incrementing primary keys, timestamps can have overlaps. But happy with it either way.